14/events/index.html

```diff
@@ -0,0 +1,1234 @@
+                        <h3>Threads</h3>
+                        <a href="http://www.qualcomm.co.in/" target=
+                        "_blank"><img alt="Qualcomm" height="50px" src=
+                        "../sponsors/img/sponsors/b_qualcomm.jpg" width=
+                        "100px"></a><br>
+                        <a class="link" href=
+                        "http://felicity.iiit.ac.in/threads/" target=
+                        "_blank">Click here for HomePage</a><br>
+                        <br>
+                        <span>Because to iterate is human, to recurse is
+                        divine</span> <a class="ca-more" href="#" id="link"
+                        name="link">more...</a>
+                    </div>
+
+
+                    <div class="ca-content-wrapper">
+                        <div class="ca-content">
+                            <a class="ca-close" href="#">close</a>
+
+                            <div class="ca-content-text" id="threads">
+                                <table>
+                                    <tr>
+                                        <td>
+                                            <h6>Online</h6>
+
+
+                                            <ul style="font-size:20px">
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href=
+                                                "http://felicity.iiit.ac.in/threads/events/codecraft"
+                                                style="width:100px" target=
+                                                "_blank">Code-Craft</a>
+                                                </li>
+
+
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href=
+                                                "http://felicity.iiit.ac.in/threads/events/tle"
+                                                style="width:100px" target=
+                                                "_blank">TLE</a>
+                                                </li>
+
+
+                                                <li style=
+                                                "padding-left:10px;padding-bottom:10px">
+                                                <a class=
+                                                "btn btn-large btn-primary"
+                                                href=
+                                                "http://felicity.iiit.ac.in/threads/events/gordian-knot"
```

Comment:
Nope again. 

@darthbhyrava 

Please fix the dangling URLs and update this PR.